content/hardware/04.pro/boards/portenta-h7/essentials.md

```diff
@@ -1,6 +1,6 @@
 ---
 productsLibrariesMap:
-  - wifi
+  - lvgl
   - arduinoble
```

Comment:
```suggestion
  - arduino_portenta_ota
  - arduinoble
```
Suggesting Arduino Portenta OTA library here, as part of relevant library that could be useful